onnxruntime/core/graph/graph.cc

```diff
@@ -3724,9 +3724,14 @@ Status Graph::ConvertInitializersIntoOrtValues() {
   std::vector<Graph*> all_subgraphs;
   FindAllSubgraphs(all_subgraphs);
 
+  const auto& model_path = GetModel().ModelPath();
+  PathString model_dir;
+  if (!model_path.empty()) {
```

Comment:
Suggest to add a comment:
When model path is empty (for example, the model proto is from memory instead of file), the model_dir will be empty, then weakly_canonical will be current working directory.